ethcore/src/state.rs

```diff
@@ -571,7 +573,21 @@ fn should_not_trace_call_transaction_to_builtin() {
 
 	let result = state.apply(&info, engine.deref(), &t, true).unwrap();
 
-	assert_eq!(result.trace, None);
+	assert_eq!(result.trace, Some(Trace {
```

Comment:
that's not a particularly great idea - i seem to remember christian putting in a lot of `CALL`s to the identity contract (0x4) in solidity, which uses it as a `memcpy`; all of these would get caught and will quite possibly severely bloat the DB.